modules/openapi-generator/src/main/resources/C-libcurl/apiClient.c.mustache

```diff
@@ -183,15 +183,10 @@ char *assembleTargetUrl(char    *basePath,
 
     int operationParameterLength = 0;
     int basePathLength = strlen(basePath);
-    bool slashNeedsToBeAppendedToBasePath = false;
 
     if(operationParameter != NULL) {
         operationParameterLength = (1 + strlen(operationParameter));
     }
-    if(basePath[strlen(basePath) - 1] != '/') {
-        slashNeedsToBeAppendedToBasePath = true;
-        basePathLength++;
-    }
```

Comment:
@ityuhui Shall we still keep the condition and just remove `slashNeedsToBeAppendedToBasePath = true;`?

Reply:
Hi @wing328 
basePathLength should not be added 1 (```basePathLength++```) because we do not append a "/" to basePath actually, but anyway, it has no harm. Just will confuse people.

Reply:
Thanks for the explanation.